features/poller/monitors/core/src/main/java/org/opennms/netmgt/poller/monitors/HttpPostMonitor.java

```diff
@@ -1,8 +1,8 @@
 /*******************************************************************************
  * This file is part of OpenNMS(R).
  *
- * Copyright (C) 2014 The OpenNMS Group, Inc.
- * OpenNMS(R) is Copyright (C) 1999-2014 The OpenNMS Group, Inc.
+ * Copyright (C) 2016 The OpenNMS Group, Inc.
```

Comment:
The "Copyright (C) " part must always contain the year when the file was created.